src/PlugKeyRing/index.ts

```diff
@@ -73,7 +73,7 @@ class PlugKeyRing {
   public deleteContact: (args: { addressName: string, subaccount?: string }) => Promise<boolean>;
   public getAgent: (args?: { subaccount ?: string, host?: string }) => HttpAgent;
   public getBalance: (args: { token: StandardToken, subaccount?: string }) => Promise<TokenBalance>;
-  public getTransactions: (args: { subaccount?: string }) => Promise<GetTransactionsResponse>;
+  public getTransactions: (args: { subaccount?: string, icpPrice: number }) => Promise<FormattedTransactions>;
```

Comment:
Why receive the icp price? Wouldnt it be better to have this on the BE directly? Would be cool to use it ln the assets as well